consumer/entertainment/estimator.go

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2021 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package entertainment
+
+const (
+	video720pMBPerMin   = 15
+	audioNormalMBPerMin = 0.75
+	browsingMBPerMin    = 0.5
+)
+
+// Estimates represent estimated entertainment
+type Estimates struct {
+	VideoMinutes    uint64
+	MusicMinutes    uint64
+	BrowsingMinutes uint64
+	TrafficMB       uint64
+	PricePerGB      float64
+	PricePerMin     float64
+}
+
+// Estimator stores average provider prices to estimate entertainment estimates
+type Estimator struct {
+	pricePerGB  float64
+	pricePerMin float64
+}
+
+// NewEstimator constructor
+func NewEstimator(pricePerGB, pricePerMin float64) *Estimator {
```

Comment:
I think our current pricing is per GiB rather than GB and you're calculating them further down the line basing on GiB.